components/SVGWebView.tsx

```diff
@@ -1,90 +1,79 @@
-import React from "react";
+import React, {useEffect} from "react";
 import {
   StyleSheet,
   TextInput,
   View,
   Dimensions,
   Animated,
-  Easing
+  ScrollView
 } from "react-native";
-
 import { Button } from "react-native-paper";
+import { SafeAreaView } from "react-navigation";
+
 import SVGMapLL from "./SVGMapLL";
 import SVGMapUL from "./SVGMapUL";
 import { MaterialIcons } from "@expo/vector-icons";
 import { colors } from "./theme";
 import theme from "./theme";
+import { searchShopsTitles } from "../helpers/filtering";
+import { LEVELS } from "../constants/constants";
+import { ShopSearchResult, NavToShopIdFunc } from "../types/types";
+import PulseButton from './PulseButton';
+
 var BGWASH = "rgba(255,255,255,0.8)";

```

Comment:
I know this isnt yours but please name it sensibly and put it in a place with the other colours

Reply:
From what I see, it is unused, so I will remove it.